src/controller/cve.controller/cve.controller.js

```diff
@@ -272,13 +277,13 @@ async function submitCna (req, res, next) {
       dateReserved: (cveId.reserved).toISOString(),
       datePublished: (new Date()).toISOString()
     }
-    const additionalProviderMetadataFields = {
-      shortName: assignerShortName
-    }
-    const cveRecord = Cve.newPublishedCve(id, cveId.owning_cna, cnaContainer, additionalCveMetadataFields, additionalProviderMetadataFields)
+
+    const providerMetadata = createProviderMetadata(orgUuid, req.ctx.org, (new Date()).toISOString())
```

Comment:
Please create a variable for the timestamp and use it for lines 277, 278 and 281 so they're all identical